### DISCUSSION TITLE
Frigate+

### DISCUSSION THREAD

**Discussion by digitalit** [blakeblackshear/frigate#6631] [Ask A Question] [answered]
Hi guys ( and maybe this is mostly for you @blakeblackshear ),

Question 1: I have an Coral USB Accelerator ( https://coral.ai/products/accelerator )  plugged into the USB-3 socket of an Intel NUC i7 with 16GB RAM running only HassOS on an SSD and i wonder if Friget+ will gain me anything besides the satisfaction of contribute financially? 

Info: Im buying a monthly subscription of Frigate+  anyway considering that I will be using frigate for the indefinite future and maybe you don't get so annoyed at my stupid questions if you know that I pay for Frigate+ every month :)

Question 2: Now when i subscribe to Frigate+ do i need to reinstall my Frigate?

Have a nice week end

**Comment by blakeblackshear** [comment, marked as answer]
The models in Frigate+ should have more accurate detections for security cameras. If you already have accuracy that is good enough, then you may not get a benefit.

In addition to being more accurate, they will have more relevant objects available to track such as packages, faces, license plates, etc. This list will expand over time.

**Reply by digitalit** [reply]
As i'm not that skilled i wonder will these models work together with my Coral USB Accelerator or in stead of my Coral USB Accelerator so i should i remove it?

**Reply by NickM-27** [reply]
they work with your coral

**Reply by digitalit** [reply]
Thank you @NickM-27 cause i'm so happy that i finally found one, it was almost impossible to find a place to buy it

**Comment by blakeblackshear** [comment]
You will not need to reinstall. Frigate+ just integrates with your existing install. https://docs.frigate.video/integrations/plus

**Reply by digitalit** [reply]
So should i use Frigate+ differently than i do now in Home Allistant?

**Reply by NickM-27** [reply]
Frigate+ is just an addition to frigate configuration, it is not a separate app / addon / etc

**Comment by digitalit** [comment]
So in my https://plus.frigate.video/dashboard/ do i have to do anything tant improves anything in Home Assistant?

I can't buy an subscription but i guess that can when models are done?

**Reply by NickM-27** [reply]
> So in my https://plus.frigate.video/dashboard/ do i have to do anything tant improves anything in Home Assistant?

Not sure I understand what you are asking

> I can't buy an subscription but i guess that can when models are done?

correct, subscription will be available when models are available

**Reply by digitalit** [reply]
Please excuse my bad spelling but it's not just my lack of the English language. I have auto correct in Swedish so some times it get very strange :)

Well, i just wondered if the https://plus.frigate.video/dashboard/ has anything to do with my cameras in HA. In HA i have a frigate.yml where i add my cameras. Is this stil the way to go or shall i ad cameras in the dashboard to use them in HA?

I guess you are laughing now, but please remember my lack of skills :)

**Reply by NickM-27** [reply]
> Well, i just wondered if the https://plus.frigate.video/dashboard/ has anything to do with my cameras in HA. In HA i have a frigate.yml where i add my cameras. Is this stil the way to go or shall i ad cameras in the dashboard to use them in HA?

This seems like two separate questions but I'll try to be clear. Frigate+ has nothing to do with HA as far as cameras in the dashboard go so no need to worry about that. As far as adding cameras in HA goes, you will want the frigate integration  https://github.com/blakeblackshear/frigate-hass-integration

**Reply by digitalit** [reply]
Well, then i'll just have to wait for the modules cause Frigate works like a charm in my HA?

![frigate_ha](https://github.com/blakeblackshear/frigate/assets/6569136/7f2f8f27-a028-4a65-8cc9-397a11123ce6)
